bpf/bpf_netdev.c

```diff
@@ -324,6 +349,13 @@ static __always_inline int handle_ipv4(struct __ctx_buff *ctx,
 		if (ret < 0)
 			return ret;
 	}
+
+#if defined(NO_REDIRECT)
+        /* Skip the stack, if we can lookup the right interface */
+        int ifindex = fib_lookup_ipv4(ip4);
+        if (ifindex > -1)
+                return redirect(ifindex, 0);
```

Comment:
Wouldn't this redirect to `cilium_host` instead of a container iface? I think we could rely on the endpoint map for redirection (see `lookup_ip4_endpoint()`) instead of `fib_lookup()`.

Reply:
This is egressing out of the container interface, no?

Reply:
This is in the `from-netdev` section, which is attached to the tc ingress. Outbound packets from the lxc should hit `to-netdev` (tc egress).